docs/appendices/release-notes/unreleased.rst

```diff
@@ -65,6 +65,9 @@ Deprecations
 Changes
 =======
 
+- Introduced new optional ```RETURNING`` clause for :ref:`Update <ref-update>` to
+  return values based on each row actually updated.
```

Comment:
```suggestion
  return values corresponding to the updated values.
```

is this accurate? I think it makes it more clear what is meant, if so

Reply:
It can be any value from the row which was updated, not just the values itself.